main.cpp

```diff
@@ -12,20 +12,28 @@
 #include <clang/Tooling/Tooling.h>
 #include <clang/Rewrite/Core/Rewriter.h>
 #include <clang/Frontend/CompilerInstance.h>
-
 using namespace clang;
 using namespace clang::ast_matchers;
 using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
+    CastCallBack(Rewriter& rewriter):r(rewriter) {
     };
 
     void run(const MatchFinder::MatchResult &Result) override {
-        // Your code goes here
+        auto E = Result.Nodes.getNodeAs<clang::CStyleCastExpr>("cast");
+        auto b = E->getLParenLoc();
+        auto e = E->getRParenLoc();
+        auto est = E->getEndLoc();
+        r.RemoveText(b,1);
+        r.InsertText(b,"static_cast<");

```

Comment:
What `Rewriter` method can be used here instead of `RemoveText()` + `InsertText()`?

Reply:
replace method